calculator_NEW.js

```diff
@@ -13,13 +13,19 @@ function accumulation(input){ //숫자, 기호 버튼 클릭 함수->acc변수
 for(var i=0;i<bt.length;i++) {
   const buttonEl = bt[i];
   buttonEl.addEventListener("click", function () {
-    accumulation(buttonEl.innerText);
+    accumulation(buttonEl.innerText); 
   });
 }
 
 function showValue(){//=버튼 클릭 함수->결과칸에 결과값 전달
   acc = eval(acc);
-  value.innerHTML = acc;
+  if(Number.isInteger(acc)===true){
```

Comment:
if (Number.isInteger(acc)) {

Reply:
이거 근데 그냥 삼항연산자 써서 하면 되는거 아닌가?
```suggestion
value.innerHTML = Number.isInteger(acc) ? acc : acc.toFixed(4)
```

Reply:
수정했음